matrix/sync.go

```diff
@@ -204,13 +224,13 @@ func (s *GomuksSyncer) OnSync(callback mautrix.SyncHandler) {
 	s.globalListeners = append(s.globalListeners, callback)
 }
 
-func (s *GomuksSyncer) notifyListeners(source mautrix.EventSource, evt *event.Event) {
+func (s *GomuksSyncer) notifyListeners(source event.Source, evt *event.Event) {
```

Comment:
In general the event source doesn't need to be passed to handlers anymore as it can be embedded in the event, but contexts should be used